src/pool.js

```diff
@@ -1,5 +1,4 @@
-import 'threads/register';
-import { Pool as tPool, spawn } from 'threads';
+import { Pool as tPool, spawn, Worker } from 'threads';
```

Comment:
I don't get the necessity of these lines, please elaborate.

Reply:
`import 'threads/register'` was there based on the documentation of threads.js and to tell `parcel-bundler` to create the worker bundle. But I discovered that it doesn't change anything, worst it prevents webpack users to build their package with geotiff.js (c.f. #143).

I've open an issue on threads.js about this subject: https://github.com/andywer/threads.js/issues/232

Reply:
Thanks for the clarification!